dao/src/main/java/greencity/entity/User.java

```diff
@@ -106,6 +106,9 @@
         inverseJoinColumns = @JoinColumn(name = "friend_id", referencedColumnName = "id"))
     private List<User> userFriends = new ArrayList<>();
 
+    @OneToMany(mappedBy = "user", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
```

Comment:
FetchType.EAGER - should be LAZY and get it in query only if it is needed

Reply:
Done